pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/Statistics.sql.stg

```diff
@@ -0,0 +1,87 @@
+group Statistics;
+
+getParticipantStatistics() ::= <<
+select
+    case when ug.user_governance_id is null then 'self' else 'governed' end as user_type,
+    count(1) as user_count
+from
+    user_study_enrollment se
+    left join umbrella_study us on se.study_id = us.umbrella_study_id
+    left join user_governance ug on ug.participant_user_id = se.user_id
+    left join user_study_governance usg on usg.umbrella_study_id = us.umbrella_study_id
+        and usg.user_governance_id = ug.user_governance_id and ug.is_active = 1
+where
+    us.umbrella_id = :studyId and
```

Comment:
`umbrella_id` is a totally different thing. This should use `umbrella_study_id` instead.

Reply:
But what is going to happen to all the data once we update the study? I thought umbrella is something that keeps the linkage between study versions. Was I wrong?

Reply:
Umbrella is a different concept. Think of it as the "organization" or "laboratory" that the study belongs to. An umbrella may own multiple different studies. Also, there is no "linkage between study versions" -- there is only one version of a study.

Reply:
Got it - changed.